CUEstion.WEB/Controllers/QuestionsController.cs

```diff
@@ -0,0 +1,283 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CUEstion.WEB.Controllers
+{
+	[ApiController]
+	[Route("[controller]")]
+	public class QuestionsController : ControllerBase
+	{
+		[HttpGet]
+		public IActionResult GetAllQuestions()
+		{
+			try
+			{
+				return Ok();
+			}
+			catch (Exception e)
+			{
+				Console.WriteLine(e);
+				return StatusCode(500, new { Message = "Server ERROR occured." });
+			}
+		}
+
+		[HttpGet("filter")]
```

Comment:
Here is the example of approach that you can use to simplify api calls with array-type parameters:
 **/lookups?options={option1}&options={option2}…&options={optionN}**

Reply:
So it will look like: **filter?tags={tag1}&tags={tag2}...** And the method signature itself will look like **GetAllQuestions(string[] tags)**